bitnami/contour/templates/envoy/daemonset.yaml

```
@@ -1,4 +1,5 @@
 {{- if .Values.envoy.enabled }}
+{{- if eq .Values.envoy.kind "DaemonSet" }}
 ---
```

Merge both conditions (also the `---` is not required):
```suggestion
{{- if and .Values.envoy.enabled (eq .Values.envoy.kind "DaemonSet") }}
```